flowauth/frontend/src/TokenDetails.js

```diff
@@ -201,16 +212,22 @@ class TokenDetails extends React.Component {
           </Typography>
         </Grid>
         <Divider />
-        <Grid item xs={12}>
+        <Grid item xs>
           <MuiPickersUtilsProvider utils={DateFnsUtils}>
             <DateTimePicker
               value={expiry}
               onChange={this.handleDateChange}
               disablePast={true}
               maxDate={latest_expiry}
+              format="yyyy/MM/dd HH:mm:ss"
+              ampm={false}
+              margin="normal"
+              helperText={new Date().toTimeString().slice(9)}
```

Comment:
Can we add a comment here that says this displays the timezone (which is not obvious otherwise)?